src/org/jitsi/impl/neomedia/format/ParameterizedVideoFormat.java

```diff
@@ -64,7 +65,7 @@ public ParameterizedVideoFormat(
         this.fmtps
             = ((fmtps == null) || fmtps.isEmpty())
                 ? MediaFormatImpl.EMPTY_FORMAT_PARAMETERS
-                : new HashMap<String, String>(fmtps);
+                : new ConcurrentHashMap<>(fmtps);
```

Comment:
The Map instance stored in the field fmtps is never modified in and exposed out of ParameterizedVideoFormat.